src/blocks/server.blocks/middleware/__router/middleware__router.node.js

```diff
@@ -56,6 +56,14 @@ modules.define(
             // remove level suffixes
             // Remove tab parts of url before processing it
             // Remove trailing slash for url
+            var tabSuffix = url.match(/\/docs\/?$/);
```

Comment:
and please add comments with an example of the string matching regexp (e.g. `// https://ru.bem.info/libs/bem-components/v2.0.0/desktop/button/examples/`)